src/schema/imported/manifest.json

```diff
@@ -897,11 +897,45 @@
         }
       }
     },
+    "GeckoAndroidSpecificProperties": {
+      "type": "object",
+      "properties": {
+        "strict_min_version": {
+          "type": "string"
+        },
+        "strict_max_version": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": {}
+    },
```

Comment:
`GeckoAndroidSpecificProperties`'s `strict_min_version`/`strict_max_version` should get a format property merged into it as for the `FirefoxSpecificProperties` type's ones.

And so we should update `src/schema/updates/manifest.json` with something like:

```diff
diff --git a/src/schema/updates/manifest.json b/src/schema/updates/manifest.json
index 014b429e..fc8cdffd 100644
--- a/src/schema/updates/manifest.json
+++ b/src/schema/updates/manifest.json
@@ -18,6 +18,18 @@
           }
         }
       },
+      "GeckoAndroidSpecificProperties": {
+        "strict_min_version": {
+          "default": "42a1",
+          "description": "Minimum version of Gecko to support. Defaults to '42a1'. (Requires Gecko 45)",
+          "pattern": "^[0-9]{1,3}(\\.[a-z0-9]+)+$"
+        },
+        "strict_max_version": {
+          "default": "*",
+          "description": "Maximum version of Gecko to support. Defaults to '*'. (Requires Gecko 45)",
+          "pattern": "^[0-9]{1,3}(\\.[a-z0-9*]+)+$"
+        }
+      },
       "ExtensionID": {
         "maxLength": 80
       },
```

And the resulting part of this diff to look like the following as a side-effect of that addons-linter schema updates:

```diff
diff --git a/src/schema/imported/manifest.json b/src/schema/imported/manifest.json
index 096b39ae..3ca33352 100644
--- a/src/schema/imported/manifest.json
+++ b/src/schema/imported/manifest.json
@@ -907,7 +907,17 @@
           "type": "string"
         }
       },
-      "additionalProperties": {}
+      "additionalProperties": {},
+      "strict_min_version": {
+        "default": "42a1",
+        "description": "Minimum version of Gecko to support. Defaults to '42a1'. (Requires Gecko 45)",
+        "pattern": "^[0-9]{1,3}(\\.[a-z0-9]+)+$"
+      },
+      "strict_max_version": {
+        "default": "*",
+        "description": "Maximum version of Gecko to support. Defaults to '*'. (Requires Gecko 45)",
+        "pattern": "^[0-9]{1,3}(\\.[a-z0-9*]+)+$"
+      }
     },
     "DeprecatedApplications": {
       "type": "object",
```

Let's also add test coverage for the gecko_android version strings formats.